packages/nbdime/src/diff/widget/notebook.ts

```diff
@@ -1,45 +1,45 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
-'use strict';
+"use strict";
```

Comment:
If we rebase this after #667, hopefully these formatting only changes will disappear :)